shell/artifacts/Products/source/ShowProduct.js

```diff
@@ -78,10 +78,9 @@ ${styles}
         <img src="{{resolvedImage}}">
       </div>
     </div>
-    <div slotid="annotation" subid="{{subId}}">
     </div>
   </div>
-  `;
+  `.trim();
```

Comment:
If you use the `html` template tag (provided by `defineParticle`) you get the `trim()` built-in and a good likelihood your editor can syntax highlight the templates.

Reply:
done